docs/advanced.rst

```diff
@@ -558,6 +558,10 @@ wherever you want, e.g.::
 
 In addition, you can also have Pipenv stick the virtualenv in ``project/.venv`` by setting the ``PIPENV_VENV_IN_PROJECT`` environment variable.
 
+☤ Custom Virtual Environment Name
+-------------------------------------
+
+Pipenv supports a custom name for the virtual environment set at ``PIPENV_CUSTOM_VENV_NAME``.  by default the venv will be named after the project directory.
```

Comment:
Would be better to say: 
```
Pipenv supports the ability to specify a custom name for the virtual environment by setting the environment variable ``PIPENV_CUSTOM_VENV_NAME`` to be whatever you want.  The logical place to specify this would be in a user's `.env` file in the root of the project, which gets loaded by pipenv when it is invoked.   The default behavior is unchanged: default name is generated by using the project directory combined with a dash and encoded hash value .
```